common/permissions.ts

```diff
@@ -0,0 +1,43 @@
+import { get, isArray } from 'lodash';
+import { PLUGIN_ID } from '.';
+
+const settings = {
+  read: 'settings.read',
+  change: 'settings.change',
+} as const;
```

Comment:
Why is `as const` explicitly needed here?

Reply:
done